src/DashboardService.Data/Interfaces/IPriorityRepository.cs

```diff
@@ -12,8 +13,10 @@ namespace LT.DigitalOffice.DashboardService.Data.Interfaces;
 public interface IPriorityRepository
 {
   Task<Guid?> CreateAsync(DbPriority dbDepartment);
-  Task<List<DbPriority>> GetAllAsync(GetPrioritiesFilter filter);
-  Task<DbPriority> GetAsync(Guid id);
-  Task<bool> EditAsync(Guid id, JsonPatchDocument<DbPriority> request);
-  Task<bool> RemoveAsync(Guid id);
+  Task<(List<DbPriority> dbPriorities, int totalCount)> GetAllAsync(GetPrioritiesFilter filter, CancellationToken ct);
+  Task<DbPriority> GetAsync(Guid id, CancellationToken ct = default);
+  Task<bool> EditAsync(Guid id, JsonPatchDocument<DbPriority> request, CancellationToken ct);
+  Task<bool> RemoveAsync(Guid id, CancellationToken ct);
+  Task<bool> NameExistAsync(string name, CancellationToken ct);
+  Task<bool> NameExistAsync(string name, Guid? priorityId, CancellationToken ct);
```

Comment:
what this method is for?

Reply:
did it by example: https://github.com/LT-Students/DigitalOffice-DepartmentService/blob/develop/src/DepartmentService.Validation/Department/EditDepartmentRequestValidator.cs#L67C45-L67C59

if another priority has same name - validation failed